core/sql/generator/GenRelUpdate.cpp

```diff
@@ -539,7 +415,9 @@ static short genHbaseUpdOrInsertExpr(
      updateRowLen,      // (OUT) Destination tuple length
      updateExpr,  // (OUT) Generated expression
      updateTupleDesc, // (OUT) Tuple descriptor for destination tuple
-     ExpTupleDesc::LONG_FORMAT);
+     ExpTupleDesc::LONG_FORMAT,
+     NULL, NULL, 0, NULL, NULL,
```

Comment:
Should this be NULL, NULL, 0, NULL, FALSE? (the last of these is an NABoolean, disableConstFolding). Though the code still works since NULL would map to FALSE.